roles/siege-install/files/siege.conf

```diff
@@ -276,7 +276,7 @@ concurrent = 25
 #
 # ex: time = 50M
 #
-# time =
+# time = 10S
```

Comment:
@bapo1 The change makes sense, but I think this line is still commented out. You need to remove the `#` comment block. :slightly_smiling_face:

Reply:
Nice catch haha, thanks